Tests/StreamChatUITests/UISDKdocumentation/ChatChannelListItemView_Documentation_Tests.swift

```diff
@@ -0,0 +1,38 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import StreamChatTestTools
+@testable import StreamChatUI
+import SwiftUI
+import XCTest
+
+class ChatChannelListItemView_Documentation_Tests: XCTestCase {
+    // Example of usage for generating UISDK documentation:
+    
+    func test_generateDocs_example_channelListItemView_namedLabelsWithPointers() {
+        // Create a view to annotate and create documentation for.
+        let view = ChatChannelListItemView().withoutAutoresizingMaskConstraints
+        
+        // Create the necessary datasource and assign it to the view.
+        let channel = ChatChannel.documentationMock(cid: .unique, unreadCount: .mock(messages: 10))
```

Comment:
Can't we just use `ChatChannel.mock` to generate the mock data?

Reply:
I wanted to differ on the actual mock, which takes time to create and create some data which can be reusable through the documentation, so we are sure we are consistent about some names or some properties to be shared across the screens, so I divided this into `documentationMock`, so there are things already prepared... I think it's faster approach than everywhere create new mock with all arguments set (because in tests we sometimes don't fill in some parameters...) I prefer this over copy-pasting....